akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/scaladsl/ClusterShardingPersistenceSpec.scala

```diff
@@ -47,7 +50,8 @@ object ClusterShardingPersistenceSpec {
       entityId,
       initialState = "",
       commandHandler = (_, state, cmd) ⇒ cmd match {
-        case Add(s) ⇒ Effect.persist(s)
+        case Add("") ⇒ Effect.invalidCommand("empty not allowed")
```

Comment:
If we go for `Confirmation`, `Accepted` and `Rejected`, it will be better to have `Effect.reject` instead. 

I have also used the word `reject` in the PR (still to be sent) I'm working on. The context was a little bit different, it is to be used when creating a command builder in the Java API. 

```
commandHandlerBuilder.rejectAny("Account already closed!")
```

In Scala, that will look more like a 
```
{
  case any => Effect.reject("Account already closed!")
}
```